spec/features/burgers_features/burger_add_spec.rb

```diff
@@ -1,17 +1,33 @@
 require "rails_helper"
 
 feature "User visits new burger page" do
+  let(:restaurant) { Restaurant.create(name: "restaurant", location: "Boston", dining_type: "Might be a selction", description: "Really long lines and really long text", hours: "10am-10pm") }
+  let(:restaurant) { FactoryGirl.create(:restaurant) }
   scenario "User sees a form to enter burger information" do
-    wendys = Restaurant.create(name: "Wendys", location: "Boston", dining_type: "Might be a selction", description: "Really long lines and really long text", hours: "10am-10pm")
-    visit new_restaurant_burger_path(wendys)
+    restaurant
+    visit new_restaurant_burger_path(restaurant)
     expect(page).to have_content "Add a burger"
     expect(page).to have_xpath "//input"
+  end
+  scenario "User successfully adds a burger" do
+    restaurant
+    visit new_restaurant_burger_path(restaurant)
 
     fill_in 'Name', with: "Double Garlic Steak Burger"
-    fill_in 'Price', with: "four dollars"
+    fill_in 'Price', with: 4
     fill_in 'Description', with: "A seasame seed bun, swiss cheese, and two beef patties slathered in Garlic Butter"
     click_button 'Add Burger'
+
+    expect(page).to have_content "Burger submitted successfully"
     expect(page).to have_current_path(burger_path(Burger.find_by(name: "Double Garlic Steak Burger")))
```

Comment:
Instead, hard code the path ("/burgers/2") you expect to see, dont be making multiple database calls in the midst of your test